src/spicedb-common/services/developerservice.ts

```diff
@@ -226,16 +226,13 @@ class DeveloperServiceRequest {
       ),
     });
 
-    const developerRequest = JSON.stringify(
-      toJson(DeveloperRequestSchema, request),
-    );
+    const developerRequest = toJsonString(DeveloperRequestSchema, request);
     const encodedResponse: string =
       window[ENTRYPOINT_FUNCTION](developerRequest);
 
-    const response = create(
-      DeveloperResponseSchema,
-      JSON.parse(encodedResponse),
-    );
+    const response = fromJsonString(DeveloperResponseSchema, encodedResponse, {
+      ignoreUnknownFields: true,
```

Comment:
`ignoreUnknownFields` is because the `fromJsonString` function was complaining that `duration` was an unknown field (down in the check debug output). It isn't used in this app, so I'm telling the function to ignore it.